File: app/src/main/java/io/github/ovso/righttoknow/data/KeyName.java
package io.github.ovso.righttoknow.data;

public enum KeyName {
  POSITION("position"),
  PAGE_TOKEN("pageToken"),
  Q("q"),
  MAX_RESULTS("maxResults"),
  ORDER("order"),
  TYPE("type"),
  VIDEO_SYNDICATED("videoSyndicated"),
  KEY("key"),
  PART("part");

  private String value;

  KeyName(String $value) {
    this.value = $value;
  }

  public String get() {
    return value;
  }
}

File: app/src/main/java/io/github/ovso/righttoknow/data/KeyName.kt
package io.github.ovso.righttoknow.data

enum class KeyName(val value: String) {
  POSITION("position"),
  PAGE_TOKEN("pageToken"),
  Q("q"),
  MAX_RESULTS("maxResults"),
  ORDER("order"),
  TYPE("type"),
  VIDEO_SYNDICATED("videoSyndicated"),
  KEY("key"),
  PART("part");

  fun get(): String {
    return value
  }

}